packages/common/src/logger/logger.ts

```diff
@@ -9,13 +9,21 @@ export interface LoggerOption {
   outputFormat?: 'json' | 'colored';
   level?: string;
   nestedKey?: string;
+  stackSize?: number;
+}
+
+function limitStack(stack: string, configSize: number): string {
+  const stackArray = stack.split(/\r\n|\r|\n/);
+  const reduceSize = Math.min(stackArray.length, configSize) + 1; //one additional line for error message
```

Comment:
The addition line can be considered in the stackSize. This `+1` is not expected by whoever using this args